src/transaction.local.net.spec.ts

```diff
@@ -62,6 +63,40 @@ describe("test transaction", function () {
         assert.deepEqual(TokenTransfer.egldFromAmount(85).valueOf(), newBalanceOfBob.minus(initialBalanceOfBob));
     });
 
+    it("should send transaction and wait for completion using the new proxy provider", async function () {
+        this.timeout(70000);
+
+        let provider = createLocalnetProvider();
+        let newProvider = new ProxyNetworkProvider("http://localhost:7950", { timeout: 5000 });
+        let watcher = new TransactionWatcher({
+            getTransaction: async (hash: string) => { return await newProvider.getTransaction(hash, true) }
```

Comment:
This snippet should be added in the PR description (and in the release notes).

Reply:
Added in the PR description. Will also add in the release nodes when it's time.